.github/workflows/acceptance.yml

```diff
@@ -102,156 +99,8 @@ jobs:
-          with:
-            path: ${{ steps.config.outputs.yarn-cache-dir }}
-            key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-            restore-keys: |
-              ${{ runner.os }}-yarn-
-
-        # setup python
-        - name: Set up Python ${{ steps.config.outputs.python-version }}
-          uses: actions/setup-python@v1
-          with:
-            python-version: ${{ steps.config.outputs.python-version}}
-
-        # setup pip
-        - name: Install pip
-          run: |
-            pip install --no-cache-dir --upgrade "pip>=20.0.2"
-
-        # pip cache
-        - name: Get pip cache dir
-          id: pip-cache
-          run: |
-            echo "::set-output name=dir::$(pip cache dir)"
-
-        - name: pip cache
-          uses: actions/cache@v1
-          with:
-            path: ${{ steps.pip-cache.outputs.dir }}
-            key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements-*.txt') }}
-            restore-keys: |
-              ${{ runner.os }}-pip-
-
-        - name: Install Javascript Dependencies
-          run: |
-            yarn install --frozen-lockfile
-
-        - name: Install Python Dependencies
-          env:
-            PGPASSWORD: postgres
-          run: |
-            python setup.py install_egg_info
-            pip install wheel # GitHub Actions does not have this installed by default (unlike Travis)
-            pip install -U -e ".[dev]"
-
-        - name: Start devservices
-          run: |
-            sentry init
-            sentry devservices up postgres redis clickhouse snuba
-
-        - name: webpack
-          run: |
-            yarn webpack --display errors-only
-
-        # Setup custom pytest matcher, see https://github.com/actions/setup-node/issues/97
-        - name: Add pytest log matcher
-          if: always()
-          run: |
-            echo "::remove-matcher owner=pytest::"
-            echo "::add-matcher::.github/pytest.json"
-
-        - name: Run acceptance tests (#${{ steps.config.outputs.matrix-instance-number }} of ${{ strategy.job-total }})
-          if: always()
-          run: |
-            mkdir -p ${{ steps.config.outputs.acceptance-dir }}
-            mkdir -p ${{ steps.config.outputs.acceptance-dir }}-mobile
-            mkdir -p ${{ steps.config.outputs.acceptance-dir }}-tooltips
-            [ "$GITHUB_REF" = "refs/heads/master" ] && export PYTEST_SENTRY_ALWAYS_REPORT=1
-            make run-acceptance
-          env:
-            PYTEST_SNAPSHOTS_DIR: ${{ steps.config.outputs.acceptance-dir }}
-            USE_SNUBA: 1
-            TEST_GROUP: ${{ matrix.instance }}
-
-        - name: Save snapshots
-          if: always()
-          uses: getsentry/action-visual-snapshot@v2
-          with:
-            save-only: true
-            snapshot-path: .artifacts/visual-snapshots
-
     py3-acceptance:
-      name: 'python3.6 acceptance'
+      name: 'python3.6'
```

Comment:
Why remove acceptance?

Reply:
Because it'll look like `acceptance / python3.6 (0)` as opposed to `acceptance / python3.6 acceptance (0)`.